doc/src/dstest-tutorial.md

```diff
@@ -0,0 +1,130 @@
+# Controlling the Unit Testing Environment
+
+## Writing Tests
+
+Test cases must be prepended with `prove_` and the testing contract must
+inherit from `Test` from (Forge's standard test
+library)[https://book.getfoundry.sh/forge/forge-std]. First, import Test:
+`import {Test} from "forge-std/Test.sol";` and then inherit from it via `... is
+Test`. This allows hevm to discover the test cases it needs to run. Like so:
```

Comment:
What about also talking about the proveFail test cases? They might be useful to some people even though you can do some workarounds with prove_ to test if it fails.

Reply:
Ah, good point, thanks! I have now added a section on this! Do see anything else missing perhaps? Any suggestions?

Reply:
Made some other pontual suggestions. I think the rest is fine. Also if needed I can help in the future on adding portuguese translated documentation :)

Reply:
Made some other pontual suggestions. I think the rest is fine. Also if needed I can help in the future on adding portuguese translated documentation :)